src/libraries/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPath.Windows.cs

```diff
@@ -677,9 +677,9 @@ public void Transform(Matrix matrix)
 
```

Comment:
On row 630-653:  For all overloads of `public void AddString(string s, FontFamily family, int style, float emSize, PointF origin, StringFormat format)` method parameters `FontFamily family, StringFormat format` are nullable

Reply:
This is true w.r.t. `StringFormat`, however in the unix implementation `FontFamily` will throw on null arguments. I think it would make sense to declare the latter as non-nullable, to avoid inconsistency in reference source generation.